tams_ur5_setup_bringup/config/apriltags2/settings.yaml

```diff
@@ -0,0 +1,14 @@
+# AprilTags 2 code parameters
+# Find descriptions in apriltags2/include/apriltag.h:struct apriltag_detector
+#                      apriltags2/include/apriltag.h:struct apriltag_family
+tag_family:        'tag36h11' # options: tag36h11, tag36h10, tag25h9, tag25h7, tag16h5
+tag_border:        1          # default: 1
+tag_threads:       2          # default: 2
+tag_decimate:      1.0        # default: 1.0
+tag_blur:          0.0        # default: 0.0
+tag_refine_edges:  1          # default: 1
+tag_refine_decode: 0          # default: 0
+tag_refine_pose:   0          # default: 0
```

Comment:
Did you test these parameters?
Better update rate is very nice but for our current demos higher precision would be even better. :)

Reply:
The flags 'tag_refine_decode' and 'tag_refine_pose' are meant to increase the number of detected tags that are otherwise too small or don't have enough contrast to be recognized.
I tested those but since our tags are clearly visible they only decrease the update rate while producing about the same accuracy and noise. I found that smoothing with higher frequency produced the best results.
I didn't test the 'tag_blur' however, as I understand it is meant as a workaround for very noisy images.